.github/workflows/hvd-tests.yml

```diff
@@ -23,15 +23,37 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+
+      - name: Get year & week number
+        id: get-date
+        run: echo "::set-output name=date::$(/bin/date "+%Y-%U")"
+        shell: bash -l {0}
+
+      - name: Get pip cache dir
+        id: pip-cache
+        run: |
+          pip3 install -U pip
+          echo "::set-output name=dir::$(python3 -m pip cache dir)"
```

Comment:
@ydcjeff should we use `python3 -m pip` or just `pip` everywhere. See for example https://github.com/pytorch/ignite/pull/1455/files#diff-3b6443c744dbe1a6a9202d29dc0c8618d7d9948b47717235541464cefefa8944R41

Reply:
Windows is missing `python3` command, if we use `python3`, mac and linux uses python2 version.
So Finalized with `pip3 cache dir` with a few warning.